pkg/generate/config/resource.go

```diff
@@ -349,3 +373,29 @@ func (c *Config) ResourceShortNames(resourceName string) []string {
 	}
 	return rConfig.ShortNames
 }
+
+// ResourceHookCode returns a string with custom callback code for a resource
+// and hook identifier
+func (c *Config) ResourceHookCode(resourceName string, hookID string) string {
```

Comment:
is this function redundant now because there is a function with similar name in hook.go file?

Reply:
Doh, nice catch :) Yes, I forgot to remote it. Doing that now...

Reply:
done.